asr1k_neutron_l3/plugins/l3/service_plugins/l3_extension_adapter.py

```diff
@@ -316,10 +321,29 @@ def create_router(self, context, router):
         asr1k_db.RouterAttsDb.ensure(context, result.get('id'))
         return result
 
+    def ensure_default_route_skip_monitoring(self, context, router_id, router):
+        tag_plugin = directory.get_plugin(TAG_PLUGIN_TYPE)
+        custom_default_route_tags = {TAG_SKIP_MONITORING, TAG_DEFAULT_ROUTE_OVERWRITE}
+
+        has_custom_default_route = any(r['destination'] == '0.0.0.0/0' for r in router['routes'])
```

Comment:
I wondered if this might be different from the check we do above, as above we only check the netmask, here we check the subnet as well, but it's probably going to be hostbit-cleaned™ anyway, so we should be fine, right?

Reply:
It typed as [AuthenticIPNetwork](https://github.com/sapcc/neutron/blob/stable/ussuri-m3/neutron/db/extraroute_db.py#L118) which in turn inherits from `netaddr.IPNetwork`. I would think it cleans the hostbit but it does not...
```
In [2]: from netaddr import IPNetwork

In [3]: str(IPNetwork('192.168.0.12/24'))
Out[3]: '192.168.0.12/24'
```
I will replace the check with `.endswith('/0')`